src/main/pg/scheduler.h

```diff
@@ -31,9 +31,13 @@
 #define SCHEDULER_RELAX_OSD 25
 #endif
 
+// Tenths of a % of tasks late
+#define SCHEDULER_LATE_LIMIT 50
+
 typedef struct schedulerConfig_s {
     uint16_t rxRelaxDeterminism;
     uint16_t osdRelaxDeterminism;
+    uint16_t cpuLatePercentageLimit;
 } schedulerConfig_t;
```

Comment:
Does the associated PG value need to be incremented?

Reply:
Don’t we leave such increments until the release to prevent them being incremented like mad during development?

So yes, but when?

Reply:
I'm not sure.
Maybe f we didn't update the PG, perhaps a user who was testing the PR and copied and pasted in a dump file from before the PR might have issues?  Or a user who didn't select full chip erase?  
Typically I've been encouraged to change it in the PR.  Unless perhaps you anticipate making more changes within the PR.

Reply:
Agree PG version needs to increased whenever the structure changes.